packages/cli/src/commands/dev.ts

```diff
@@ -52,9 +52,12 @@ async function storeDev() {
 
 export default class Dev extends Command {
   async run() {
+    const basePath = process.cwd()
+
+    const { getRoot, tmpDir } = withBasePath(basePath)
 
     const queueChange = (/* path: string, remove: boolean */) => {
-      generate()
+      generate({ basePath })
```

Comment:
question: the basePath option is optional on the generate function, and when it's not passed it uses `process.cwd()`. I'm wondering why you chose to pass it here and some other places, but the line 79 of this file wasn't updated for example, is there a reason?

Reply:
I wanted to pass on all the places, but I probably forgot line 79. The reason I'm passing it everywhere is to make the next PR even simpler (the one that will introduce behavior change).